site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -46,8 +49,13 @@ export const UsersPage: React.FC = () => {
 
   // Fetch users on component mount
   useEffect(() => {
-    usersSend("GET_USERS")
-  }, [usersSend])
+    const filter = searchParams.get("filter")
+    const query = filter !== null ? filter : userFilterQuery.active
```

Comment:
```suggestion
    const query = filter ?? userFilterQuery.active
```

Reply:
Thanks for the suggestion, added it in two places. 😄